.github/workflows/ci.yml

```
@@ -74,6 +74,6 @@ jobs:
     needs: build-and-test
     with:
       core_branch: ${{ github.ref }}
-      petals_branch: develop
+      petals_branch: add-lxml-dependency  # TODO: DO NOT MERGE LIKE THIS
```

Revert once https://github.com/CLIMADA-project/climada_petals/pull/168 is merged
```suggestion
      petals_branch: develop
```